src/App.js

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
+import { Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Navbar from './pages/Navbar';
+import Welcome from './pages/Welcome';
+import Quote from './pages/Quote';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = '';
-  }
-
-  render() {
-    return <Calculator />;
-  }
-}
+const App = () => (
+  <>
+    <Navbar />
+    <Routes>
+      <Route exact path="/home" element={<Welcome />} />
```

Comment:
- [ ] when starting up the application the user gets an empty page initially this is due to the fact that the root path in your code is `/home` if you change is to `/` in both the App.js file and the Navbar.js file it will start the application on your welcome page.

Reply:
Thanks for this observation. Changes are going to be made accordingly